deltadb/src/deltadb.c

```diff
@@ -461,7 +464,7 @@ struct deltadb * deltadb_create( const char *logdir )
 
 	if(logdir) {
 		db->logdir = strdup(logdir);
-		log_recover(db,time(0));
+		log_recover(db,timestamp);
 	}
 
 	return db;
```

Comment:
After this: `update` and `remove` should be a no-op if the object was created as a snapshot.